appcelerator/titanium_mobile
# feat(ios,android): expose Ti.UI.Slider „tintColor“ and „trackTintColor“

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-27272

**Summary:**
Added new properties "tintColor" and "trackTintColor" to `Ti.UI.Slider`.

---
**Test 1 - Set Colors Upon Creation:**
1. Build and run the below on iOS.
2. Verify that the left side of the slider is red.
3. Verify that the right side of the slider is green.
4. Repeat test on Android.

```javascript
var window = Ti.UI.createWindow({
	backgroundColor: "white",
});
window.add(Ti.UI.createSlider({
	min: 0.0,
	max: 1.0,
	value: 0.5,
	tintColor: "red",
	trackTintColor: "green",
}));
window.open();
```

---
**Test 2 - Set Colors After Creation:**
1. Build and run the below code on iOS.
2. Tap on the "Change Slider Color" button.
3. Verify left side of the slider bar changed to red.
4. Verify right side of the slider bar changed to green.
5. Repeat test on Android.

```javascript
var window = Ti.UI.createWindow({
	backgroundColor: "#CCCCCC",
});
var slider = Ti.UI.createSlider({
	min: 0.0,
	max: 1.0,
	value: 0.5,
});
window.add(slider);
var button = Ti.UI.createButton({
	title: "Change Slider Color",
	bottom: "30dp",
});
button.addEventListener("click", function() {
	slider.tintColor = "red";
	slider.trackTintColor = "green";
});
window.add(button);
window.open();
```
